scripts/compile-kernel.sh

```diff
@@ -101,16 +114,16 @@ case "$COMPILER" in
 			--rm \
 			-v "$KERNEL_SRC_DIR:/linux/" \
 			kernel-gcc \
-			/bin/sh -c "cd linux && make clean && make $KERNEL_CONFIG && make -j32"
+			/bin/sh -c "cd linux && rm -f all.err && make clean && make $KERNEL_CONFIG && make -j32 $CHECKER"
 		;;
 	clang)
 		docker run \
 			--rm \
 			-v "$KERNEL_SRC_DIR:/linux/" \
 			kernel-clang \
-			/bin/sh -c "cd linux && \
+			/bin/sh -c "cd linux && rm -f all.err && \
```

Comment:
I would place the `rm -f all.err` command in the same line as the `make clean`, because `rm -f all.err` is more like `make clean` in its intention than the `cd linux` command.

Reply:
done